lib/header.c

```diff
@@ -68,12 +68,6 @@ static const int typeSizes[16] =  {
     0
 };
 
-enum headerSorted_e {
-    HEADERSORT_NONE	= 0,	/* Not sorted */
-    HEADERSORT_OFFSET	= 1,	/* Sorted by offset (on-disk format) */
-    HEADERSORT_INDEX	= 2,	/* Sorted by index  */
-};
```

Comment:
Why are we getting rid of the enum? It seems we're still using the magic values, shouldn't we keep the names so we know what they're for?

Reply:
Hmm? There are no "magic values" left anywhere, h->sorted is now a simple boolean to indicate whether the header is sorted or not, as the variable name indicates (and was actually misleading before).

Reply:
Oh, in some intermediate version the commit message pointed to commit da3a3a14e757ccd517e2eb2a3f0293ff48b3ff7f for background, seems that I've edited it out at some point. I can certainly add a remark of it now becoming a real boolean for the first time.

Reply:
Added a remark about h->sorted to the commit message, hopefully clearing this part up.